code/modules/goonchat/browserassets/css/browserOutput.css

```diff
@@ -274,12 +274,12 @@ em						{font-style: normal;	font-weight: bold;}
 
 .say					{}
 .deadsay				{color: #e2c1ff}
-.binarysay				{color: #1e90ff;}
+.binarysay				{color: #20c20e;	background-color: #000000;	display: block;}
```

Comment:
This shouldn't be part of your PR.

Reply:
I know i mentioned it in coderbus but no one replied. Sadly I'm not exactly sure how to remove that part from the PR request.